src/containers/Review/ReviewWrapper.tsx

```diff
@@ -22,6 +22,8 @@ const ReviewWrapper: React.FC<ReviewWrapperProps> = ({ structure }) => {
     userState: { currentUser },
   } = useUserState()
 
+  // I think we need an option to selecte review assgnments where
+  // userId is reviewerId or assignerId or both or not match it at all (just by row level permission restrictions)
```

Comment:
I agree. So we don't need to send the current user here. Although when displaying in the UI we need to show different from what ReviewRow is yours and what isn't. Plus, how to set if the Reviewer name is visible or not in the UI, based on permission?

Reply:
Hmm, do we need to say 'assigned to other' vs assigned to 'name' ?

I was imagining that in the expansion we show review_assignments that are 'yours' (reviewer id or assigner). And similar in home page, but there is a toggle to switch between 'yours' and visible to you. How does that sound ?